### PR TITLE
Feature/add scope to sampling context asgi integration

### PR DESCRIPTION
**Pull request by Lynaj**
Inside of the `traces_sampler` method, I need an access to the `path`, as well as to the `method`. These are stored inside of the `scope`.
It helps me to filter out URLs, that I don't want to include in the performance report.

### CODE REVIEW

**Comment by untitaker** [comment]
I don't know the context of this so I will have to defer.

**Comment by Lynaj** [comment]
@rhcarvalho @lobsterkatie can I somehow help with the review?

File: tests/integrations/asgi/test_asgi.py [hunk 1]
```diff
@@ -202,3 +206,42 @@ def handler(*args, **kwargs):
...
+):
+    test_url = "/sync-message"
+
+    traces_sampler = mock.Mock()
+    sentry_init(
+        send_default_pii=True,
+        traces_sampler=traces_sampler
+    )
+
+    events = capture_events()
```
Comment (untitaker) [inline tests/integrations/asgi/test_asgi.py:224]: can you do something with this var, such as asserting the event did/did not get sent?

**Comment by lobsterkatie** [comment]
@Lynaj - sorry for the delay. This is actually part of a larger change I was working on, which I just merged: https://github.com/getsentry/sentry-python/pull/906.

Please let me know if that doesn't serve your needs.

**Pull request closed without merging**